go/worker/common/worker.go

```diff
@@ -339,7 +346,7 @@ func New(
 ) (*Worker, error) {
 	cfg, err := newConfig()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "worker/common: failed to initialize config")
```

Comment:
For new code, no need to use the external errors package (see #2057):
```suggestion
		return nil, fmt.Errorf("worker/common: failed to initialize config: %w", err)
```